src/main/kotlin/org/rust/lang/core/psi/ext/RsFunction.kt

```diff
@@ -80,6 +80,27 @@ val RsFunction.returnType: Ty get() {
 
 val RsFunction.abi: RsExternAbi? get() = externAbi ?: (parent as? RsForeignModItem)?.externAbi
 
+/**
+ * A function is unsafe if defined with `unsafe` modifier or if defined inside `extern` block.
```

Comment:
It's a little bit misleading because not all function from extern block are unsafe